admin/audit.md

```diff
@@ -58,3 +58,75 @@ actions included:
 - `view`: the Coder CLI used a secret
 - `write`: the user made a change to a Coder entity (e.g., workspace, user,
   resource pool, etc.)
+
+## Admin logged events
+
+With the exception of a few, logged events made by Admin panel changes will output
```

Comment:
Should we document the exception?

Reply:
we should, but those will be added soon, so we'd have to go back and remove it. i'm thinking whatever is not documenting isn't logged. what do you think?

Reply:
that's fair.